src/els_text_synchronization.erl

```diff
@@ -19,9 +19,10 @@ did_save(Params, Server) ->
   Uri          = maps:get(<<"uri">>         , TextDocument),
   CDiagnostics = els_compiler_diagnostics:diagnostics(Uri),
   DDiagnostics = els_dialyzer_diagnostics:diagnostics(Uri),
+  EDiagnostics = els_elvis_diagnostics:diagnostics(Uri),
   Method = <<"textDocument/publishDiagnostics">>,
   Params1  = #{ uri => Uri
-              , diagnostics => CDiagnostics ++ DDiagnostics
+              , diagnostics => CDiagnostics ++ DDiagnostics ++ EDiagnostics
```

Comment:
Diagnostics are becoming complex. Should we allow users to customize their behaviour via the `erlang_ls.config` file? E.g. I may want to disable elvis, but keep compiler diagnostics.

Reply:
I agree, for a follow-up PR.

And we should also consider what happens if we have a diagnostics source that takes longer to run, should we send some results when we have them, or only send them all when the slowest one finishes.